src/components/common/convert-json/convert-json.tsx

```diff
@@ -1,47 +1,48 @@
 import SanitizedHtml from './sanitize-html'
 
+function isJson(inputString: string): boolean {
+  let value = typeof inputString !== 'string' ? JSON.stringify(inputString) : inputString
+  try {
+    value = JSON.parse(value)
+  } catch (err: unknown) {
+    console.debug('Invalid Json.', err)
+    return false
+  }
+
+  return typeof value == 'object' && value !== null
+}
+
 export function convertWysiwygToComponent(inputString: string) {
   if (inputString === '') {
     return <></>
   }
 
-  // Check if it is draftJs or HTML
-  try {
+  if (isJson(inputString)) {
     const parsedJson = JSON.parse(inputString)
-
-    if (parsedJson['entityMap']) {
-      return (
-        <>
-          {parsedJson.blocks.map(
-            (
-              block: {
-                inlineStyleRanges: any[]
-                text: string
-                key: string
-              },
-              index: string
-            ) => (
-              <p
-                key={block.key + index}
-                style={{
-                  fontWeight: block.inlineStyleRanges.some((range) => range.style === 'BOLD') ? 'bold' : 'normal',
-                }}
-              >
-                {block.text}
-              </p>
-            )
-          )}
-        </>
-      )
-    }
-  } catch (error) {
-    if (error instanceof SyntaxError && error.message.includes('JSON.parse')) {
-      // inputString is HTML instead of DraftJs Json
-      return <SanitizedHtml htmlString={inputString}></SanitizedHtml>
-    }
-
-    // else log the errors
-    console.error(error)
-    return <></>
+    return (
+      <>
+        {parsedJson.blocks.map(
```

Comment:
We probably want to safeguard against the possibility of `blocks` being undefined here. A conditional return that checks for `parsedJson?.blocks`, using an `if` statement or a ternary operator perhaps.

Reply:
Good point. I'll updated the code with a conditional `?` operator.